src/mainScanner.py

```diff
@@ -103,7 +103,7 @@ def scan(ports, range_low, range_high):
 
     # Printing the information to screen
     print 'Scanning Completed in: ', total
-    return render_template('index.html', portnum=portnum, host=remoteServerIP, range_low=range_low, range_high=range_high, total=total)
+    return render_template('input.html', portnum=portnum, host=remoteServerIP, range_low=range_low, range_high=range_high, total=total)

```

Comment:
Why `input.html`? Can't we have `index.html`?

Reply:
Sir, I as we have two routes in our app <domain>/ and <domain>/input, so better making two separate files as per the content to be shown on these routes (we don't want to have the empty RESULTS of port scan on <domain>/ route).

Reply:
@NetanMangal You had to use Hide/Show toggle feature using JavaScript in the client Side. This is what @vinitshahdeo meant.
> _P.S. this was an interesting solution to the issue._